core/mgmt/v1beta/metric.proto

```diff
@@ -29,9 +29,9 @@ enum Status {
 
 // ========== Pipeline endpoints
 
-// PipelineTriggerCount represents a pipeline execution count with some
+// TriggerCount represents a execution count with some
```

Comment:
nit: Nothing against this change but then we should perhaps move the message from the `Pipeline endpoints` section.